src/app-host/app-host.js

```diff
@@ -1,11 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
-
+/* global io:false */
 var livereload = require('./live-reload-client');
 var Messages = require('messages');
 var telemetry = require('telemetry-helper');
 
 var cordova;
-var oldExec;
 var socket = io();
```

Comment:
what about using io as `window.io`, and it is clear that is a global window variable, in the context of the browser? then we don't need to ignore the rule.